packages/jest-config/jest.config.ts

```diff
@@ -0,0 +1,38 @@
+import type { Config } from '@jest/types';
+import deepmerge from 'deepmerge';
+
+const baseConfig: Config.InitialOptions = Object.freeze({
+  preset: 'ts-jest',
+  transform: {
+    // Enable JS/JSX transformation
+    '\\.(ts|js)x?$': 'ts-jest',
+    '\\.(svg)$': 'jest-transform-stub',
+  },
+  transformIgnorePatterns: [
+    '[/\\\\]node_modules[/\\\\]((?!ky[/\\\\]).)+\\.(js|jsx|mjs|cjs|ts|tsx)$',
```

Comment:
nit: ky ignorePatterns is exclusive for ky import projects should not include in shared jest-config files?

Reply:
Thought about that, since it's only a shared pattern/rule it's ok to leave it? Compare to eslint rules, you may not going to write some code that triggers the rule, but the rule will help you when it happens.